license-bill-of-materials.go

```diff
@@ -217,13 +208,8 @@ func expandPackages(gopath string, pkgs []string) ([]string, error) {
 	cmd.Env = fixEnv(gopath)
 	out, err := cmd.CombinedOutput()
 	if err != nil {
-		output := string(out)
-		if strings.Contains(output, "cannot find package") ||
-			strings.Contains(output, "no buildable Go source files") {
```

Comment:
The output of the go tool changed since this check was written. Just going to remove the custom error type.